nixos/modules/services/web-apps/matomo.nix

```diff
@@ -65,23 +72,27 @@ in {
             (import ../web-servers/nginx/vhost-options.nix { inherit config lib; })
```

Comment:
Yes, somehow I really managed to get this done. :smile: If you try to merge with `//` operator it doesn't work as this would discard my custom defaults as soon as the user defines something custom. After a lot of fiddling around, I discovered `recursiveUpdate`, which does exactly what I need here, together with the `locations` definitions and the well-dosed `mkForce` at `https://github.com/NixOS/nixpkgs/pull/34028/files#diff-8c50bc76feff6200b2774e17f7f44c52R199`.

I also tried to achieve that via `mkDefault` etc., but I could not find a way that worked. Which does not mean there isn't, but I don't understand enough of the priority attribute merging system to find it…